executor/runtime/docker/docker.go

```diff
@@ -2140,14 +2140,20 @@ func (r *DockerRuntime) k8sContainerToDockerConfigs(c *runtimeTypes.ExtraContain
 	if err != nil {
 		return nil, nil, nil, fmt.Errorf("Parsing the liveness probe for the %s container: %w", v1Container.Name, err)
 	}
+	entrypoint := computeExtraContainersDockerEntrypoint(c)
+	command := computeExtraContainersDockerCommand(c)
+	if len(entrypoint) == 0 && len(command) == 0 {
+		err := fmt.Errorf("An Entrypoint or Command must be specified for the container")
```

Comment:
Nice, thank you for this! Would this error show up in the UI next to the specific container which failed, or should it container the name of the container with missing data?

Reply:
The caller will wrap this and name the container that failed to create